CMakeLists.txt

```diff
@@ -48,8 +60,13 @@ target_include_directories(edlib_static PUBLIC
 add_executable(helloWorld apps/hello-world/helloWorld.c)
 target_link_libraries(helloWorld edlib_static)
 
-add_executable(runTests test/runTests.cpp)
-target_link_libraries(runTests edlib_static)
+include(CTest)
```

Comment:
What is the point of this CTest, how does that help or improve something?

Reply:
`CTest` implicitly adds the *standard* CMake option `ENABLE_TESTING`, which we can use to disable building unit tests. Some people don't want to run and build tests, and having a standardized flag for it means we don't have to second guess hwo to disable tests. Finally, this creates a standard CMake target `test`, which Gentoo and other distros use to run tests, instead of having to manually invoke `./runTests` or something, which helps us automate building.

Reply:
Cool, that sounds great!